server/routes/api/browser.py

```diff
@@ -142,6 +145,13 @@ def get_statvar_group():
     stat_var_group = request.args.get("stat_var_group")
     places = request.args.getlist("places")
     result = dc.get_statvar_group(stat_var_group, places)
+    if os.environ.get('FLASK_ENV') == 'production':
```

Comment:
can you put this in configmodule.py, so we have a central place to record the config.

Reply:
what config are we recording? the environment?

Reply:
done